src/pages/invitee/EventInvite.jsx

```diff
@@ -94,17 +128,18 @@ const EventInvite = () => {
               <input
                 name="preferred_date_time"
                 type="datetime-local"
-                placeholder="17/11/2022 - 3pm"
-                value={inviteDetails.date}
+                value={inviteDetails.preferred_date_time ? inviteDetails.preferred_date_time : preferredTime}
                 onChange={changeInviteDetails}
                 className="outline-none border border-[#898989] rounded md:w-[477px] w-full px-3 py-3 text-base font-medium"
+                min={startDate}
+                max={endDate}
```

Comment:
- [ ] Kindly implement date range validation such that a user can only select within the range of the preferred host start date and end date

**Hint**: Something similar was implemented in the create event preferred date. Only the selected start date and end date was visible when the host wanted to select his preferred date